utilities/src/main/java/cz/xtf/openshift/imagestream/ImageRegistry.java

```diff
@@ -191,6 +191,10 @@ public String tcpProxy() {
 		return normalize(TestConfiguration.get().readValue(TestConfiguration.IMAGE_TCP_PROXY));
 	}
 
+	public String oracleDriver() {
+		return this.normalize(TestConfiguration.get().readValue("xtf.oracle-driver"));
```

Comment:
This breaks consistency in approach we are handling the images atm... Please add corresponding values also to `TestConfiguration` class.